docs/introduction.rst

```diff
@@ -115,28 +115,22 @@ capture sensor information and other data that can be retrieved using IPMI.
 In RackHD these become “pollers” that periodically capture telemetry data from
 the hardware interfaces.
 
-What RackHD is good at
+What RackHD Does Well
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The obvious mechanism of provisioning an OS is one of the more straightforward
-workflows you can image, and fundamentally RackHD is really focused on being
-the lowest level of automation supporting interrogating hardware, setting a
-“personality” onto it (in the form of an OS), providing consistent REST based
-API’s for controlling that hardware - agnostic of hardware vendor, and for
-using the pollers to capture telemetry - create “live data feeds” that can be
-provided via a pub/sub style interface.
-
-As we went through use cases and expanded features, we made the capability for
-the workflow engine to react to what it discovered - what we call “SKU” support,
-dynamic rendering of templates for OS installs, and passing of variables and
-data from the APIs that invoke workflows through to the configuration files that
-drive OS installs - like a kickstart or debseed file.
-
-While we have a number of workflows in our code repository as examples of how
-you can do a variety of actions, the real power of the system is in being able
-to create your own workflows - and submit those through the REST API. So you
-can define arbitrary workflows for your needs, specific to your hardware if
-needed, to accomplish your automation goals.
+RackHD is focused on being the lowest level of automation that interrogates agnostic hardware,
+provisions machines with operating systems. With SKU support, the workflow engine can react to
+information found during discovery and respond with dynamic rendering of installation templates.
+During this process, the API can be used to pass variables and data to the configuration files
```

Comment:
Drop the "during this process" - as it implies (to me) that when the workflow is running you can interact with it via an API call to pass in additional information, which you can't. 

Maybe instead "The API can be used to pass in data through variables in the workflow configuration, so you can parameterize your workflows. The workflows also have access to all of the SKU information and other catalogs, so workflows can be authored that react to that information"